pkg/controller/m3admin_client.go

```diff
@@ -221,3 +221,7 @@ func (c errorPlacementClient) Add(instance placementpb.Instance) error {
 func (c errorPlacementClient) Remove(id string) error {
 	return c.err
 }
+
+func (c errorPlacementClient) Replace(_ string, _ placementpb.Instance) error {
```

Comment:
why do these parameters have underscores?

Reply:
It's a thing I've seen in Go before to indicate the parameter isn't used in the function, but I just realized I can drop the `_` entirely! Going to do that